src/queue.rs

```diff
@@ -37,7 +37,6 @@ impl Queue {
 
     /// Push a node to the tail end of the queue.
     pub(crate) fn push(&self, node: Node) {
-        node.enqueue();
```

Comment:
what did this previously?

Reply:
This toggled a flag in the node that enqueued it, but that flag is no longer used.